docs/source/en/tasks_explained.mdx

```diff
@@ -0,0 +1,230 @@
+
+Ready to try your hand at audio classification? Check out our complete [audio classification guide](tasks/audio_classification) to learn how to finetune wav2vec2 and use it for inference!
+
+### Automatic speech recognition
+
+To use the pretrained model for automatic speech recognition, add a language modeling head on top of the base wav2vec2 model for [connectionist temporal classification (CTC)](glossary#connectionist-temporal-classification-ctc). The language modeling head is just a linear layer that accepts hidden state output and performs a linear transformation to convert them into logits. Each logit represents a token class (the number of tokens comes from the task vocabulary), and the CTC loss is calculated between the logits and target to find the most likely token.
+
+Ready to try your hand at automatic speech recognition? Check out our complete [automatic speech recognition guide](tasks/asr) to learn how to finetune wav2vec2 and use it for inference!
+
+## Computer vision
+
+There are two ways to approach computer vision tasks:
+
+1. Use a modern CNN, like ConvNeXT, which relies on convolutional layers but adopts new network designs.
+2. Use a Transformer architecture to split an image into patches and process the patches as a sequence.
+
+<Tip>
+
+A third approach is a hybrid Transformer with convolutions (for example, [Convolutional Vision Transformer](model_doc/cvt), or [LeViT](model_doc/levit)). We won't discuss those here because they just combine the two approaches we examine here.
+
+</Tip>
+
+ViT and ConvNeXT are commonly used for image classification, but for other vision tasks like object detection, segmentation, and depth estimation, we'll take a look at DETR and GLPN, respectively; these models are better suited for those tasks.
+
+### Image classification
+
+ViT and ConvNeXT are both used for image classification; the main difference is that ViT uses a Transformer architecture while ConvNeXT uses convolutions.
+
+#### Transformer
+
+[ViT](model_doc/vit) replaces convolutions entirely with a pure Transformer architecture. If you're familiar with the original Transformer, then you're already most of the way toward understanding ViT.
+
+The main change ViT introduced was in how images are processed:
+
+1. An image is split into square patches. If you had a 224x224 pixel image, you could split it into 256 14x14 image patches. Just like how text is tokenized into words, an image is "tokenized" into a sequence of patches. Each patch is flattened by concatenating the height and width pixel values into a single vector. For a 3D image, the height, width, and number of *channels* are flattened.
```

Comment:
I think you can shorten this paragraph by only mentioning that "Each patch is then flattened into a single vector." This is likely enough to know here.